modular_skyrat/master_files/code/modules/clothing/towels.dm

```diff
@@ -15,6 +15,10 @@
 #define TOWEL_WORN_ICON 'modular_skyrat/master_files/icons/mob/clothing/towel.dmi'
 /// Icon path to the worn icon of the towel for digitigrades.
 #define TOWEL_WORN_ICON_DIGI 'modular_skyrat/master_files/icons/mob/clothing/towel_digi.dmi'
+/// Icon path to the left-hand inhand icons of the towel.
+#define TOWEL_LEFTHAND_ICON 'modular_skyrat/master_files/icons/mob/inhands/clothing/towel_lefthand.dmi'
```

Comment:
I just realized that I never added `#undef` for any of these in the past, I probably should go back and do that. I wouldn't ask that from you in this PR, though.